src/main/java/eu/xenit/gradle/docker/alfresco/internal/version/AlfrescoVersion.java

```diff
@@ -27,8 +29,9 @@
 
     @Nullable
     public static AlfrescoVersion fromAlfrescoWar(@Nonnull Path warPath) throws IOException {
-        Path versionPropertiesPath = warPath;
-        for(String pathComponent: VERSION_PROPERTIES_PATH) {
+        FileSystem zipFs = FileSystems.newFileSystem(warPath, null);
```

Comment:
Got this from SonarCloud:
https://sonarcloud.io/project/issues?branch=addWar-as-smartCopy&id=xenit-eu_alfresco-docker-gradle-plugin&issues=AXKVj067HFpu0oNBlWU6&open=AXKVj067HFpu0oNBlWU6

zipFs should probably be closed in a finally block